packages/sdk/src/Base.ts

```diff
@@ -495,6 +495,7 @@ class Base {
       if (txOptions.gasPrice?.lt(MinPolygonGasPrice)) {
         txOptions.gasPrice = BigNumber.from(MinPolygonGasPrice)
       }
+      txOptions.gasLimit = BigNumber.from(300e3)
```

Comment:
I think this value should be a constant `PolygonMinGasLimit`

Also are we certain that all polygon transaction won't exceed 300k gas limit? I had originally thought there was going to be an additional buffer added to the estimated gas limit but I know that may be more complicated so if we know that 300k gas limit is enough for all polygon txs then we can merge